packages/animations/pubspec.yaml

```diff
@@ -16,3 +16,15 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   vector_math: ^2.1.0
+
+screenshots:
+  - description: 'Thumbnail of a container transform animation.'
+    path: example/demo_gifs/container_transform_thumbnail.webp
```

Comment:
I am not super in-love with how this thumbnail looks. But i also don't have a better idea of what to put here, so LGTM.

Reply:
I can see what you mean. Hopefully it is good enough to encourage users to trigger the image carousel.